cert-manager/config/rendered/release.yaml

```diff
@@ -0,0 +1,338 @@
+apiVersion: v1
```

Comment:
this file should not be put into  git but rather it should be in the `.gitignore`